src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -8233,6 +8233,22 @@ void M()
             await VerifyItemExistsAsync(markup, "Bar");
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ExtensionMethodWithinSameClassOfferedForCompletion()
+        {
+            var markup = @"
+public static class Test
+{
+    static void TestB()
+    {
+        T$$
```

Comment:
Nit, you don't need to include the "T"; the test harness will automatically test what's displayed with nothing typed, with the first character typed, and with the whole thing typed.